src/plugins/expressions/common/ast/build_function.ts

```diff
@@ -183,8 +183,10 @@ export function buildExpressionFunction<
       acc[key] = value.map((v) => {
         return isExpressionAst(v) ? buildExpression(v) : v;
       });
-    } else {
+    } else if (value !== undefined) {
```

Comment:
Would you mind adding a unit test for this case? Since it wasn't something I had thought about previously